content/blog/2020-09-28-september-20-community-gems.md

```diff
@@ -0,0 +1,198 @@
+---
+title: September '20 Community Gems
+date: 2020-09-28
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  customizing your DVC plots, the difference between external dependencies
+  and outputs, and how to save models and data in continuous integration.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  customizing your DVC plots, the difference between external dependencies
+  and outputs, and how to save models and data in continuous integration.
+picture: 2020-09-28/Gems_Sept_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/september-20-community-gems/512
+tags:
+  - Discord
+  - Gems
+  - CML
+  - Hyperparameters
+  - External
+  - SSH
+  - Vega
+---
+
+## DVC questions
+
+### [Q: When I try to push to my DVC remote, I get an error about my SSH-RSA keys. What's going on?](https://discordapp.com/channels/485586884165107732/485596304961962003/748735263634620518)
+
+If you're using DVC with an SSH-protected remote, DVC uses a Python library
+called `paramiko` to create a connection to your remote. There is a
+[known issue](https://stackoverflow.com/questions/51955990/base64-decoding-error-incorrect-padding-when-loading-putty-ppk-private-key-to)
+that `paramiko` expects RSA keys in OpenSSH key format, and can throw an error
+if the keys are in an alternative format (such as default PuTTY formatted keys).
+If this is the case, you'll likely see:
+
+```
+ERROR: unexpected error - ('... ssh-rsa ...=', Error('Incorrect padding',))
+```
+
+To fix this, convert your RSA key to the OpenSSH format. Tools like PuTTYgen and
+MobaKeyGen can help you do this.
+
+### [Q: Can I have multiple `param.yaml` files in a project?](https://discordapp.com/channels/485586884165107732/563406153334128681/753322309942509578)
+
+Yes, you can have as many separate parameter files as you'd like. It's only
+important that they are correctly specified in your DVC pipeline stages.
+
+For example, if you have files `params_data_processing.yaml` and
+`params_model.yaml` in your project (perhaps to store hyperparameters of your
+data processing and model fitting stages, respectively), you'll want to call the
+right file at each stage. For example:
+
+```dvc
+dvc run -n preprocess \
+  -p params_data_process.yaml:param1,param2,...
+```
+
+### [Q: Is there a way to automatically produce SVG plots from `dvc plot`? I don't like having to click through the Vega-Lite GUI to get an SVG, and my plots look so small when I access them in the browser.](https://discordapp.com/channels/485586884165107732/563406153334128681/750012082149392414)
```

Comment:
Looks like two question? Should we split them?

Reply:
I think this works as one, would rather keep it together since the two parts are so related. It's just about two different ways to handle customizing your plots (always go straight to SVG, or keep as html and fix size)

Reply:
👍